app/controllers/candidate_interface/restructured_work_history/job_controller.rb

```diff
@@ -44,7 +44,11 @@ def destroy
 
       current_application.update!(work_history_completed: false)
 
```

Comment:
If it's the last job, should we also be destroying the work history breaks? Otherwise, we end up with a borked ui